chart/values.schema.json

```
@@ -463,6 +463,21 @@
                             "type": "boolean",
                             "default": false
                         },
+                        "cacheSize": {
+                            "description": "Add cache size for your statsD deployment.",
+                            "type": "integer",
+                            "default": true
```

As the value is of type int I assume the default should be numeric. in the values.yaml you have 1000.
```suggestion
                            "default": 1000
```

---

made changes, thanks!